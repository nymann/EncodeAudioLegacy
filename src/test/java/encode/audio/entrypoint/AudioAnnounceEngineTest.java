package encode.audio.entrypoint;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.DiffReporter;
import org.approvaltests.reporters.UseReporter;
import org.junit.Assert;
import org.junit.Ignore;
import org.junit.Test;

@UseReporter(DiffReporter.class)
public class AudioAnnounceEngineTest {

    @Test
    public void checkJUnit() {
        Assert.assertTrue(true);
    }

    @Test
    public void checkApprovalTestsReporter() throws Exception {
        Approvals.verify("My diff reporter open a new window of my default diff tool. It works!");
    }
}
