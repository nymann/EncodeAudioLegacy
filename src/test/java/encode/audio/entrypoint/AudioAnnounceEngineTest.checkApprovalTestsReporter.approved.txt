My diff reporter open a new window of my default diff tool. It works!